js/blog.js

```diff
@@ -0,0 +1,70 @@
+//Async request to JSON file for Blog post
+const getPosts = async () => {
+  const response = await fetch("../js/Blog.json");
+  const data = response.json();
+  return data;
+};
```

Comment:
We don't need to make an API call to the blog's file. You can directly import the object from the .json file.